.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
       jdk: oraclejdk8
       before_install: nvm install 8
       script:
-        - cd src && npm i && npm run tsc && cd ../demo
+        - cd src && npm i && npm run tsc && npm run postclone gitHubUsername=TheGitHubUser pluginName=ThePlugin initGit=y includeTypescriptDemo=y includeAngularDemo=n && cd ../demo
```

Comment:
Actually the postclone script has `npm i` in it so we do not need anything to have install since the first and only script that is required to be executed first is `npm run postclone`. I will remove this from the travis